src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -207,6 +209,17 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
 
     }
 
+    def processBakingAndEndorsingRights(fetchingResults: tezosNodeOperator.BlockFetchingResults): Future[List[Unit]] = {
```

Comment:
I can't see a reason to have a `List[Unit]` vs. `Unit`, maybe it would be a better return type?

Reply:
yes, it is, changed